make/Makefile-local-install

```diff
@@ -0,0 +1,82 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~ LOCAL DIST INSTALL SCRIPTS ~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# ~~~~~~~~~~~ INSTALL DEPENDENCY SCRIPT ~~~~~~~~~~~~~~
+
+# Install system requirements for MDAI
+# .SILENT: mdai-system-reqs
+.PHONY: mdai-system-reqs
+mdai-system-reqs:
+  @echo "🟢 Start mdai-system-reqs..."
+  @go version || brew list go || brew install go \
+  kubectl version || brew list kubectl || brew install kubectl \
+  npm -v || brew list npm || brew install npm \
```

Comment:
Why is `npm` needed? I don't see it used, but maybe some other command is using it?

Reply:
good call out, `npm` is needed for AWS (`cdk`) but not local install